profile/README.md

```diff
@@ -0,0 +1,24 @@
+<p align="center">
+  <img src="https://github.com/ipfs/.github/blob/main/profile/ipfs-powered.png" width="100%" alt="IPFS Powered">
+</p>
+
+The InterPlanetary File System (IPFS) is a set of composable, peer-to-peer protocols for addressing, routing, and transferring [content-addressed data](https://docs.ipfs.tech/concepts/glossary/#content-addressing) in a decentralized file system. Many popular Web3 projects are built on IPFS - see the [ecosystem directory](https://ecosystem.ipfs.tech/) for a list of some of these projects.
+
+Exceptionally talented open communities are building the future of the decentralized web. There are excellent "good first issues" that need your attention and support! Join the movement, get involved today!
```

Comment:
Do we want to link to `good first issue` only or `help wanted` too? I included both but in the search link suggested here but maybe `good first issue` would be a better curated starting point?

```suggestion
Exceptionally talented open communities are building the future of the decentralized web. Excellent [good first issues](https://github.com/search?q=org%3Aipfs+label%3A%22good+first+issue%22%2C%22help+wanted%22&type=issues) need your attention and support! Join the movement! Get involved today!
```